wagtail_streamfield_migration_toolkit/autodetect/questioner.py

```diff
@@ -10,3 +10,7 @@ def ask_block_rename(self, old_path, new_path):
     def ask_block_remove(self, old_path):
         msg = "Was '{}' removed? [y/N]"
         return self._boolean_input(msg.format(old_path))
+
+    def ask_block_not_changed(self, old_path):
```

Comment:
Suggestion: personally I would name these functions like `ask_if_same_block`, `ask_if_block_removed` etc., I think things are easier to read if they are more like spoken phrases.